front/tailwind.config.js

```diff
@@ -76,10 +76,22 @@ module.exports = {
             filter: "brightness(80%)",
           },
         },
+        "cursor-blink": {
+          "0%": {
+            opacity: 1,
+          },
+          "90%": {
+            opacity: 1,
+          },
```

Comment:
Why do you need the 90%?

Reply:
I need it to blink for only a 10th of the time it is actually displayed. Otherwise the text moves, the cursor moves and blink for too long and we don't understand what is going on from UI standpoint.